client/src/app/pages/applications/components/application-detail-drawer/application-detail-drawer.tsx

```diff
@@ -117,9 +117,20 @@ export const ApplicationDetailDrawer: React.FC<
                 <ApplicationBusinessService
                   id={application.businessService.id}
                 />
-              ) : null}
+              ) : (
+                t("terms.unassigned")
+              )}
+            </Text>
+            <Title headingLevel="h3" size="md">
+              {t("terms.migrationWave")}
+            </Title>
+            <Text component="small">
+              {application?.migrationWave
+                ? application.migrationWave.name
+                : t("terms.unassigned")}
```

Comment:
Is this part of the fix for MTA-1382?  I don't see anything in the jira about migration waves.